Sources/ParseSwift/LiveQuery/Subscription.swift

```diff
@@ -60,7 +60,8 @@ private func == <T>(lhs: Event<T>, rhs: Event<T>) -> Bool {
 /**
  A default implementation of the `ParseSubscription` protocol. Suitable for `ObjectObserved`
  as the subscription can be used as a SwiftUI publisher. Meaning it can serve
- indepedently as a ViewModel in MVVM.
+ indepedently as a ViewModel in MVVM. Also provides a publisher for pull responses of query such as:
+ `find`, `first`, `count`, and `aggregate`.
```

Comment:
This Oxford comma is really triggering me but all our docs are in American English so I really can't complain 😖😅